public/app/features/datasources/components/picker/DataSourcePicker.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import {
+  DataSourcePicker as DeprecatedDataSourcePicker,
+  DataSourcePickerProps as DeprecatedDataSourcePickerProps,
+} from '@grafana/runtime';
+import { config } from 'app/core/config';
+
+import { DataSourcePickerWithHistory } from './DataSourcePickerWithHistory';
+import { DataSourcePickerWithHistoryProps } from './types';
+
+type DataSourcePickerProps = DeprecatedDataSourcePickerProps | DataSourcePickerWithHistoryProps;
```

Comment:
@ivanortegaalba in what way has the DataSourcePickerProps defined in runtime changed? (this is exposed to plugins)